docs/platform/release-notes/tizen-6-0-m1.md

```diff
@@ -0,0 +1,692 @@
+  - Libraries are updated by gmp 6.1.2, mpc 1.1.0, mpfr 4.0.2, and isl 0.22.
+  - Cilk runtime is dropped in upstream.
+- Glibc (GNU 2.30), Binutils (GNU 2.33.1) have been upgraded.
+  - The NIS (+) support library, libnsl, has been also deprecated from glibc 2.26.
+  - Libio.h has been removed from glibc 2.28.
+- Compilation options have been updated.
+  - The default mode for C++ is now -std=gnu++14 instead of -std=gnu++11.
+  - Program Instrumentation Option (-fstack-protector-strong) and Linker option (-Wl,-z,relro) have been added globally to improve security.
+  - Code generation Option (-frecord-gcc-switches) has been added and now used command line is recorded into the object file as .GCC.command.line.
+  - "_FILE_OFFSET_BITS=64" has been added as default on armv7l.
+  - New warning options have been enabled by default in -Wall -Wextra switches.
+- Toolchain testing infrastructure has been enabled.
+  - Packages (autogen, dejagnu, guile, libgc, tcl and expect) for toolchain testing have been added. 
+  - “{package}-testresults.rpm” in gdb, binutils, glibc, and gcc have been provided with run_tests macro.
+
+
+## Machine Learning
+
+### New and changed features
+
+- ML/Inference API
+  - Pipeline .NET API set has been added.
+  - Single C/.NET API set has been updated.
+    - Single API support for flexible input tensor dimensions has been added.
+      - Applications are no longer required to close and re-open a handle to change input tensor dimensions.
+    - Single API support for hardware acceleration configurations has been added.
+    - Tizen’s Neural Network Runtime (nnfw) has been integrated.
+    - Performance enhancement: low-latency open and invocation.
+    - Timeout for invocation may be specified.
+- NNStreamer update
+  - NNStreamer is upgraded from 1.0.0 to 1.5.2.
+  - Tizen Sensor Framework may be used as stream inputs.
+  - AI acceleration hardware and neural network framework support has been added.
+    - ML API now allows controlling HW acceleration.
+    - ML API may utilize NCSDK2, Edge-TPU, OpenVINO, ARMNN, Caffe2, and PyTorch.
+      - You may need to download and install additional adaptors from tizen.org for these additional frameworks or hardware.
+  - The ability to insert filters implemented as a C++ class (OpenCV compatible) or a simple C function has been added.
+- nnfw: Neural Network Runtime
+  - nnfw has been upgraded to version 1.4.0.
+  - Support for quantized MobileNet and Inception V3 models on CPU Backend has been added.
+
+### Fixes
+
+- Neural network Single API set (5.5 M2 known issue) / dimension changes
+  - New Single API set to support flexible tensor dimensions has been added.
+- Inference API set / additional neural networm framework supports
+  - Tizen’s Neural Network Runtime (nnfw) has been integrated.
+  - Others (TensorFlow, Caffe2, PyTorch, and so on) have been supported optionally if the device has additional packages downloaded and installed from tizen.org.

```

Comment:
@myungjoo,
Thank you for incorporating all the comments.

Request you to please remove the extra space before and after the "/":

- Neural network Single API set (5.5 M2 known issue)/dimension changes
  - New Single API set to support flexible tensor dimensions has been added.
- Inference API set/additional neural networm framework supports

Reply:
@surbhig12 / @wtprince  Such changes appear good to me as well. Thanks.